packages/vsce/src/trees/treeItems/CICSLibraryTreeItem.ts

```diff
@@ -58,35 +58,28 @@ export class CICSLibraryTreeItem extends TreeItem {
 
     this.children = [];
     try {
-      https.globalAgent.options.rejectUnauthorized = this.parentRegion.parentSession.session.ISession.rejectUnauthorized;
 
       const libraryResponse = await getResource(this.parentRegion.parentSession.session, {
         name: "cicslibrarydatasetname",
         regionName: this.parentRegion.getRegionName(),
         cicsPlex: this.parentRegion.parentPlex ? this.parentRegion.parentPlex.getPlexName() : undefined,
         criteria: criteria,
       });
-      https.globalAgent.options.rejectUnauthorized = undefined;
-      const datasetArray = Array.isArray(libraryResponse.response.records.cicslibrarydatasetname)
-        ? libraryResponse.response.records.cicslibrarydatasetname
-        : [libraryResponse.response.records.cicslibrarydatasetname];
-      this.label = `${this.library.name}${this.parentRegion.parentPlex ? ` (${this.library.eyu_cicsname})` : ""}${
-        this.activeFilter ? ` (${this.activeFilter}) ` : " "
-      }[${datasetArray.length}]`;
+      const datasetArray = toArray(libraryResponse.response.records.cicslibrarydatasetname);
+      this.label = `${this.library.name}${this.parentRegion.parentPlex ? ` (${this.library.eyu_cicsname})` : ""}${this.activeFilter ? ` (${this.activeFilter}) ` : " "
+        }[${datasetArray.length}]`;
```

Comment:
this is a bit tricky and the whitespace has changed with no real benefit to the reader.

what about something like this? i guess the linter still won't like it, but...

```suggestion
      const datasetArray = toArray(libraryResponse.response.records.cicslibrarydatasetname);
      this.label = `${this.library.name}
        ${this.parentRegion.parentPlex ? ` (${this.library.eyu_cicsname})` : ""}
        ${this.activeFilter ? ` (${this.activeFilter}) ` : " "}
        [${datasetArray.length}]`;
```

Reply:
There are similar examples elsewhere, we could choose a common approach (whatever it is).

Reply:
Had a go at resolving in [this commit](https://github.com/zowe/cics-for-zowe-client/pull/173/commits/37980ec8493398fbc434908bc1b893630905bc41) - what do you think?

Reply:
I think that is way easier to maintain - guarantee it would take several goes to get the syntax, brackets, and backticks lined up any time I changed the previously implementation.